indico/modules/events/logs/templates/entry_simple.html

```diff
@@ -34,4 +34,14 @@
             </td>
         </tr>
     {% endfor %}
+    {% if data_query %}
+        <tr class="i-table content">
+            <td class="i-table caption log-caption"></td>
+            <td class="i-table value">
+                <a href="{{ url_for('.index', entry.event, **data_query) }}">
```

Comment:
I think this should be done in the react app, not in the template. That way you:

- avoid the full page reload
- integrate nicely with the rest of the UI

Maybe a "Related log entries" button between previous/next in the modal? Unless you have a better idea...